### PR TITLE
feat: kosmosctl support using the install and uninstall command

### PR DESCRIPTION
**Pull request by ONE7live**

### CODE REVIEW

File: pkg/kosmosctl/install/install.go [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+	ctlutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/i18n"
+
+	"github.com/kosmos.io/kosmos/pkg/kosmosctl/manifest"
+	"github.com/kosmos.io/kosmos/pkg/kosmosctl/util"
+	"github.com/kosmos.io/kosmos/pkg/version"
+)
+
+const (
+	clusterlinkSystem         = "clusterlink-system"
```
Comment (duanmengkk) [inline pkg/kosmosctl/install/install.go:26]: should it be kosmos-system

Comment (ONE7live) [reply pkg/kosmosctl/install/install.go:26]: well~considering the current stability，I will modify this after other modules are installed and accessed @duanmengkk

**Pull request merged**